### PR TITLE
[Maintenance] Updating stub profile

### PR DESCRIPTION
**Pull request by Mxrcon**
Hey @abhi18av I'd like to suggest the addition of a standard container (ubuntu 20.04) on the stub profiles to make possible to stub run using k8s structure. 

## PR content
* changes on stub profile

## Ending
Feel free to make any question and discuss those changes with me, I'd like to implement them on our other pipelines if you agreed with them.
Hope we have a merge!

### CODE REVIEW

**Comment by abhi18av** [comment]
@Mxrcon , I thought we agreed on the pattern of combining `stub.conf` and `stub_params.yml` for keeping the number of files used for testing to be minimal - is there something that I'm forgetting?

File: conf/stub.config [hunk 1]
```diff
@@ -1,14 +1,133 @@
...
+    withName:
+    'FASTQC' {
+        container = "gliderlabs/alpine"
+        cpus = 1
+        memory = "2 GB"
+    }
+
     withName:
-    ".*" {
-        cpus = 2
```
Comment (abhi18av) [inline conf/stub.config]: @Mxrcon , perhaps there has been a misunderstanding. We don't need to specify the process names at all. We simply needed to add a `process.container = "ubuntu:20.04"` to the previous `stub.config`.

**Review by abhi18av — COMMENTED** [review]
Added my doubts about a possible mis-understanding 🤔

**Comment by Mxrcon** [comment]
ohh true!

> @Mxrcon , I thought we agreed on the pattern of combining `stub.conf` and `stub_params.yml` for keeping the number of files used for testing to be minimal - is there something that I'm forgetting?

That's true I misunderstood it, I need to aply this pattern to the other pipelines not to this one!

>  We don't need to specify the process names at all. We simply needed to add a process.container = "ubuntu:20.04" to the previous stub.config

Yes! I would like to know if you like this explicit or you prefer to keep it minimal as it was before,

**Comment by abhi18av** [comment]
😃 maintainence troubles 😉 

TIP: When a review comment is added, try to revert back within that thread itself.

**Comment by Mxrcon** [comment]
ok! I'll re-focus now that the misunderstand is solved, and make prs on our other branches

**Review by abhi18av — APPROVED** [review]
Looks good to merge 👍 

Thanks @Mxrcon 😊

**Pull request merged**